assignment/js/main.js

```diff
@@ -1,3 +1,192 @@
 /* =====================
  Copy your code from Week 4 Lab 2 Part 2 part2-app-state.js in this space
 ===================== */
+
```

Comment:
If `main.js` is loaded in at the bottom of the HTML, does jQuery need to be wrapped in $(document).ready()?